sharkfuser/templates/external_tools.h.in

```diff
@@ -0,0 +1,2 @@
+// Auto-generated from external_tools.h.in - do not edit
+#define IREE_COMPILE_PATH "@SHARKFUSER_EXTERNAL_IREE_COMPILE@"
```

Comment:
For my understanding - does the `@SHARKFUSER_EXTERNAL_IREE_COMPILE@` get materialized by CMake when it runs the `configure_file` above?